### PR TITLE
SPE-2008: Add medical accountability scorecard

### PR DESCRIPTION
**Pull request by JamesJedi420**
## Summary

- Adds pure `buildMedicalAccountabilityScorecard` composer (`src/domain/medicalAccountabilityScorecard.ts`) that deterministically folds upstream **finding arrays** from SPE-2010 staff treatment telemetry, SPE-2003 medical outcome deviation audit, and SPE-2006 treatment-failure blame routing into scorecard rows, eight first-slice finding kinds, summary counts, and audit-facing lines.
- Uses **type-only** imports from upstream modules; does not call upstream report builders or recompute upstream logic.
- Adds 23 focused Vitest cases in `src/test/medicalAccountabilityScorecard.test.ts`.

**Linear:** https://linear.app/spectranoir/issue/SPE-2008/batch-9977-ideological-medicine-scorecard-with-outcome-accountability

**Upstream dependencies:** SPE-2010, SPE-2003, SPE-2006 (merged on main via #2140)

**Smallest boundary:** pure composer + tests only.

## Files changed

- `src/domain/medicalAccountabilityScorecard.ts`
- `src/test/medicalAccountabilityScorecard.test.ts`

## Validation run

- `npm run test:run -- src/test/medicalAccountabilityScorecard.test.ts`
- `npm run test:run -- src/test/staffTreatmentTelemetry.test.ts`
- `npm run test:run -- src/test/medicalOutcomeDeviationAudit.test.ts`
- `npm run test:run -- src/test/treatmentFailureBlameRouting.test.ts`
- `npm run test:run` (full suite: 3170 passed)
- `npm run lint`
- `git diff --check`

## Gap / edge-case / boundary audit

1. **Scorecard correctness:** Consumes upstream findings only; conservative grouping (subject+protocol+week, staff+week, site+week, aggregate); all eight finding kinds implemented; missing site scores stay undefined (not zero); accountability route quality documented (neutral 50 with simple deltas).
2. **Dependency correctness:** Type-only upstream imports; no reverse imports; no GameState/UI.
3. **Determinism / mutation:** Sorted rows/findings/lines; inputs not mutated; duplicate site `signalId` deduped by first sorted entry.
4. **Boundary check:** No persistence, tick, UI, doctrine enforcement, care-mode selector, policy ledger, or upstream reimplementation.
5. **Documentation:** Module header only; no backlog/planning edits.

## Out of scope

- GameState persistence, weekly tick, runtime integration
- UI / developer overlay
- Doctrine enforcement, care-mode selector, policy ledger
- Blame-routing / deviation / telemetry implementation
- Real-world diagnosis or protected-class encoding
- SCP / franchise strings in generated output

## Test plan

- [x] Empty input → empty report
- [x] Each upstream source builds rows
- [x] Each scorecard finding kind + summary alignment
- [x] Determinism, immutability, threshold overrides, line hygiene

### CODE REVIEW

**Comment by JamesJedi420** [comment]
@copilot apply changes based on the comments in [this thread](https://github.com/JamesJedi420/containment-protocol/pull/2141#pullrequestreview-4303909783)

File: src/domain/medicalAccountabilityScorecard.ts [hunk 1]
```diff
@@ -0,0 +1,999 @@
...
+      (finding) =>
+        finding.kind === 'high_alignment_low_efficacy' ||
+        finding.kind === 'outcome_below_expected'
+    )
+  ) {
+    return DEFAULT_OPTIONS.highAlignmentThreshold
+  }
+
+  return undefined
+}
```
[the remark this answers is not in the document]
Comment (JamesJedi420) [reply src/domain/medicalAccountabilityScorecard.ts:415]: Addressed in 77b4434: fixed coarse inference (85/30) for kind-only rows; configured thresholds apply in buildRowFindings only.

File: src/domain/medicalAccountabilityScorecard.ts [hunk 2]
```diff
@@ -0,0 +1,999 @@
...
+
+  const outcomeDeviationCount = deviationFindings.filter((finding) =>
+    OUTCOME_DEVIATION_KINDS.has(finding.kind)
+  ).length
+  if (outcomeDeviationCount > 0) {
+    return Math.max(0, DEFAULT_OPTIONS.poorOutcomeThreshold - 5 * (outcomeDeviationCount - 1))
+  }
+
+  return undefined
+}
```
[the remark this answers is not in the document]
Comment (JamesJedi420) [reply src/domain/medicalAccountabilityScorecard.ts:459]: Addressed in 77b4434: same coarse-inference approach; threshold overrides honored at finding emission.

File: src/domain/medicalAccountabilityScorecard.ts [hunk 3]
```diff
@@ -0,0 +1,999 @@
...
+    accommodationAccessScore: siteScores.accommodationAccessScore,
+    careModeCoverageScore: siteScores.careModeCoverageScore,
+    governanceReviewPressureScore: deriveGovernanceReviewPressureScore({
+      deviationFindings: bucket.deviationFindings,
+      blameFindings: bucket.blameFindings,
+      governanceReviewCapacityScore: siteScores.governanceReviewCapacityScore,
+    }),
+  }
+}
+
```
[the remark this answers is not in the document]
Comment (JamesJedi420) [reply src/domain/medicalAccountabilityScorecard.ts:633]: Addressed in 77b4434: options drive buildRowFindings threshold checks; row derivation stays threshold-independent by design.

File: src/domain/medicalAccountabilityScorecard.ts [hunk 4]
```diff
@@ -0,0 +1,999 @@
...
+    getOrCreateBucket(buckets, siteWeekKey(signal.siteId, signal.week)).siteSignals.push(signal)
+  }
+
+  const rows = [...buckets.values()]
+    .map((bucket) => buildRow(bucket))
+    .sort(compareRows)
+
+  const findings = [...buckets.values()]
+    .flatMap((bucket) => {
+      const row = rows.find((candidate) => candidate.rowId === bucket.rowId)
```
[the remark this answers is not in the document]
Comment (JamesJedi420) [reply src/domain/medicalAccountabilityScorecard.ts]: Addressed: single-pass bucket loop builds row and findings together (O(n)).

File: src/domain/medicalAccountabilityScorecard.ts [hunk 5]
```diff
@@ -0,0 +1,999 @@
...
+    }
+  }
+
+  const buckets = new Map<string, RowBucket>()
+
+  for (const finding of staffFindings) {
+    const bucket = getOrCreateBucket(buckets, resolveStaffFindingKey(finding))
+    if (bucket.staffId === undefined && finding.staffId.trim().length > 0) {
+      bucket.staffId = finding.staffId.trim()
+    }
```
[the remark this answers is not in the document]
Comment (JamesJedi420) [reply src/domain/medicalAccountabilityScorecard.ts]: Addressed in 77b4434: resolveBucketStaffId() with sorted unique IDs; mixed-staff rows omit staffId. Order-independence test added.
